cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -186,7 +188,7 @@ func (p *BicepProvider) EnsureEnv(ctx context.Context) error {
 
 			p.env.DotenvSet(environment.ResourceGroupEnvVarName, rgName)
 			if err := p.envManager.Save(ctx, p.env); err != nil {
-				return fmt.Errorf("saving resource group name: %w", err)
+				return fmt.Errorf("saving location to env: %w", err)
```

Comment:
Should this be reverted? Seems to be setting the resource group not location

Reply:
https://github.com/Azure/azure-dev/pull/5032